tracker/tracker/src/main/app/nodes.ts

```diff
@@ -57,6 +65,26 @@ export default class Nodes {
     }
     return id;
   }
+  cleanTree() {
+    // sadly we keep empty items in array here resulting in some memory still being used
+    // but its still better than keeping dead nodes or undef elements
+    // plus we keep our index positions for new/alive nodes
+    // performance test: 3ms for 30k nodes with 17k dead ones
+    let cleaned = 0;
```

Comment:
is this cleaned variable temporary till the PR is finalized?

Reply:
Yeah I used it for the tests. Its removed now.